tests/tests/util/options.rs

```diff
@@ -245,13 +251,25 @@ impl ImageComparison {
 pub struct RenderOptions {
     optional: bool,
     sample_count: u32,
+    exclude_warp: bool,
 }
 
 impl Default for RenderOptions {
     fn default() -> Self {
         Self {
             optional: false,
             sample_count: 1,
+            exclude_warp: false,
         }
     }
 }
+
+impl RenderOptions {
+    pub fn is_supported(&self, adapter: &wgpu::Adapter) -> bool {
+        let info = adapter.get_info();
+        if self.exclude_warp && cfg!(windows) && info.vendor == 5140 && info.device == 140 {
```

Comment:
Perhaps an explanation, or a link, explaining where these magic values came from, or what they mean, would be nice here.